scripts/linters/pre_commit_linter.py

```diff
@@ -261,6 +264,11 @@ def _get_linters_for_file_extension(
             verbose_mode_enabled=verbose_mode_enabled)
         third_party_linters.append(third_party_linter)
 
+    elif file_extension_to_lint == 'proto':
+        custom_linter, third_party_linter = proto_linter.get_linters(
+            _FILES['.proto'], verbose_mode_enabled=verbose_mode_enabled)
+        third_party_linters.append(third_party_linter)
```

Comment:
I'm not sure which files these linters will lint, are we going to have *.proto file in the future? Will that be auto-generated or we are going to write it?

It would be great if you can explain this in brief. [It will be helpful for me to maintain these new checks!]

Reply:
Actually, in initial design we had plans for `.proto` files in Oppia but later on we moved them to an independent third party repo. So this linter is no longer required (and I don't think it will be required in future either). Hence, this will be removed.

Reply:
Removed proto_linter.

Reply:
Awesome! Thanks for the info.